### DISCUSSION TITLE
having difficulty connecting to Anki on macOS

### DISCUSSION THREAD

**Discussion by paj80paj** [jasonwilliams/anki#76] [Q&A] [answered]
nice extension. I expect it to be very useful. 
unfortunately right now I get the following error: 

Deck not sent: request to http://127.0.0.1:8765/ failed, reason: connect ECONNREFUSED 127.0.0.1:8765

my markdown looks like this

# new deck title

## first question

here is a bunch of text that's not very useful and is the answer to my first question

## second question

here is some more text it's not very meaningful blah blah blah

**Comment by jasonwilliams** [comment, marked as answer]
@paj80paj silly question but is Anki running? And do you have AnkiConnect installed?

**Reply by paj80paj** [reply]
I didn't realize that you needed Ankiconnect and it works fine now

**Reply by paj80paj** [reply]
thank you for a prompt answer to an Anki Newbie
